packages/components/src/internal/util/messaging.tsx

```diff
@@ -57,9 +57,19 @@ export function resolveErrorMessage(error: any, noun = 'data', nounPlural?: stri
             lcMessage.indexOf('violation of unique key constraint') >= 0 ||
             lcMessage.indexOf('cannot insert duplicate key row') >= 0
         ) {
-            return `There was a problem ${verb || 'creating'} your ${noun || 'data'}.  Check the existing ${
-                nounPlural || noun
-            } for possible duplicates and make sure any referenced ${nounPlural || noun} are still valid.`;
+            let match = errorMsg.match(/.+, (.*)\) already exists./);
```

Comment:
We do support using comma in sample name. What would the message be for "S, 1"?

Reply:
This pattern works just fine as long as you don't have a comma and a space, but needs to be less greedy when you have both (someone surely will).  Updated.